index.html

```diff
@@ -59,6 +60,214 @@ <h3 class="connect">LET'S CONNECT</h3>
             </li>
             
           </ul>
+        </section>
+
+            <!--Card One-->
+        <section class="section-1">
+
+          <div class="first-card " >
+            <div class="items">
+              <div>
+                <img src="./images/Snapshoot Portfolio.png" alt="">
+              </div>
+
+              <div>
+                <h1>Tonic</h1>
+              </div>
+
+              <div class="details">
+                <h3>CANOPY</h3>
+                <img src="./images/Counter.png" alt="">
+                <h4>Back End Dev</h4>
+                <img src="./images/Counter.png" alt="">
+                <h4>2015</h4>
+              </div>
+
+              <p>
+                A daily selection of privately<br>personalised reads; no accounts or<br>sign-ups required.
+              </p>
+
+              <ul class="languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+              </ul>
+          
+              <button class="see-project"><a href="#">
+                See Project</a>
+              </button>
+            </div>
+          </div>
+
+          <!--Card Two-->
+          <div class="first-card">
+            <div>
+              <img src="./images/Snapshoot Portfolio (2).png" alt="">
+            </div>
+
+            <div>
+              <h1>Multi-Post<br>Stories</h1>
+            </div>
+
+            <div class="details">
+              <h3>CANOPY</h3>
+              <img src="./images/Counter.png" alt="">
+              <h4>Back End Dev</h4>
+              <img src="./images/Counter.png" alt="">
+              <h4>2015</h4>
```

Comment:
- [ ] Kindly ensure that the `alt` attribute has the meaningfull description. Alt text is critical for those with disabilities because it describes the function and appearance of a photo or graphic they cannot see.